### DISCUSSION TITLE
The best way to tag template and remove when clone.

### DISCUSSION THREAD

**Discussion by Another5405** [SilentVoid13/Templater#997] [Help] [answered]
I want to use tag to manage template,

I did try <% tp += " #Tag "%> or <% tR += " #Tag "%>
both remain the Tag in clone.

Have any way to solve this?

Custom function will be my last option cause I may use multi vault.

Thankyou.

**Comment by welpdx** [comment, marked as answer]
I think you are looking for 
```js
<%* tR += " #Tag " %>
```

tR is to output the text
when using tR add an asterisk to `<%`

for more info, read here: https://silentvoid13.github.io/Templater/commands/execution-command.html#how-to-output-a-value-from-a-javascript-execution-command-

Alternatively, use:
```js
<% " #Tag " %>
```

both would ouput " #Tag "

**Reply by Another5405** [reply]
Thank You , I don't realize the way make no bug that is don't define a string!

**Reply by Another5405** [reply]
I did try <% " #Tag "%>,
yes it output text,but i want to remove the tag when make a clone so:
<%* if (" #Template " !=null) { %> <%* } %> is my solution,
Anyway, thank you for help.
